bundles/org.openhab.binding.sonnen/src/main/resources/OH-INF/i18n/sonnen.properties

```diff
@@ -16,6 +14,8 @@ thing-type.config.sonnen.sonnenbattery.refreshInterval.label = Refresh Interval
 thing-type.config.sonnen.sonnenbattery.refreshInterval.description = How often in seconds the sonnen battery should schedule a refresh after a channel is linked to an item. Valid input is 0 - 1000.
 thing-type.config.sonnen.sonnenbattery.authToken.label = Authentication Token
 thing-type.config.sonnen.sonnenbattery.authToken.description = Authentication Token which can be found under "Software Integration" if you connect locally to your sonnen battery. If empty the old deprecated API will be used.
+thing-type.config.sonnen.sonnenbattery.chargingPower.label=sonnen battery maximum charging power
```

Comment:
As a reminder: when the labels in the xml are updated, this file needs to be regenerated

Reply:
ok